hyperopt/spark.py

```diff
@@ -49,7 +49,7 @@ class SparkTrials(Trials):
     # Hard cap on the number of concurrent hyperopt tasks (Spark jobs) to run. Set at 128.
     MAX_CONCURRENT_JOBS_ALLOWED = 128
 
-    def __init__(self, parallelism=None, timeout=None, spark_session=None):
+    def __init__(self, parallelism=None, timeout=None, loss_threshold=None, spark_session=None):
```

Comment:
as recommended in the dev guidelines, please use black to ensure you don't exceed the 88 chars